### DISCUSSION TITLE
Android 13 support

### DISCUSSION THREAD

**Discussion by jamespole** [solokeys/solo2#163] [General]
Since upgrading my Samsung S21 phone to Android 13 I seem to have lost the ability to use my Solokey USB-C key. Websites that request a security key pops up a Passkey dialog which doesn't seem to support external USB security keys. My Solokey works fine in other devices e.g. Edge/Windows on a HP laptop and Firefox/Linux on a Lenovo laptop. As the Solokey works fine on other devices I am inclined to suspect my Android phone as the culprit somehow. There is limited info on this but I have found a post in Google's support forums with the same trouble with Yubikeys on the S21. Has as anyone else had this experience with the S21 or any other Android 13 phone? Am keen to rule out any user error on my part.

**Comment by dpatterbee** [comment]
I think I'm having the same issue with a Google Pixel 6 on Android 13. When I attempt to use a passkey it pops up the Google passkey prompt, with no option of using a hardware key, only a passkey saved in your Google account (or on another device or something idk).

I'm also assuming that this is Google fucking everything up, but maybe somebody knows some way around this?

**Reply by msiqx9** [reply]
I was setting up a new OnePlus phone (Android 13.1) yesterday and also experienced this problem with a Yubico USB-C/NFC key.
